File: c#/QuitCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


    class QuitCommand : Command
    {
        public QuitCommand(Game g)
            : base(g)
        {
        }

        public override void Execute()
        {
            this.game.Running = false;
        }
    }

File: c++/adventuregame/QuitCommand.cpp
//=================================
//C++ OOP
//Name: Katariina Pohjola
//Student id: 1201141
//Assignment: the adventure game 6
//==================================
#include "Command.h"
#include "Game.h"
using namespace std;
////////////////////////////////////////////////////////////////////////////////
//quit command implemention
void QuitCommand::Execute()
{
	GetGame()->SetRunning(false);
}
////////////////////////////////////////////////////////////////////////////////

